utils/rowgroup/rowgroup.h

```diff
@@ -1329,7 +1329,8 @@ inline void Row::setVarBinaryField(const uint8_t* val, uint32_t len, uint32_t co
       len = getColumnWidth(colIndex);
 
     *((uint16_t*)&data[offsets[colIndex]]) = len;
-    memcpy(&data[offsets[colIndex] + 2], val, len);
+    if (val != nullptr)
```

Comment:
IMHO, if we compare val with null ptr the else should log this.
We could replace this if with idbassert but it doesn't look good to assert so logging suits better IMHO.

Reply:
As you can see in line 1316, nullptr is expected here. I've added assert here ` idbassert(val != nullptr || !len);`